python/hsfs/feature_descriptive_statistics.py

```diff
@@ -0,0 +1,201 @@
+#
+#   Copyright 2023 Hopsworks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import json
+import humps
+from typing import Optional, Mapping
+from hsfs.util import FeatureStoreEncoder
+
+
+class FeatureDescriptiveStatistics:
+    def __init__(
+        self,
+        min: float,
+        max: float,
+        sum: float,
+        count: int,
+        mean: float,
+        stddev: float,
+        completeness: Optional[float] = None,
+        num_non_null_values: Optional[int] = None,
+        num_null_values: Optional[int] = None,
+        distinctness: Optional[float] = None,
+        entropy: Optional[float] = None,
+        uniqueness: Optional[float] = None,
+        approx_num_distinct_values: Optional[int] = None,
+        exact_num_distinct_values: Optional[int] = None,
+        percentiles: Optional[Mapping[str, float]] = None,
+        id: Optional[int] = None,
+    ):
+        self._id = id
```

Comment:
Might want to add a field with the feature data_type?

Reply:
Makes sense